src/assets/room_devices/pet_bowl/pet_bowl.asset.ts

```diff
@@ -0,0 +1,47 @@
+DefineRoomDeviceAsset({
+	name: 'Pet bowl',
+	size: 'medium',
+	colorization: {
+		bowl: {
+			name: 'Pet bowl',
+			default: '#FFFFFF',
+		},
+		interior: {
+			name: 'Pet bowl interior',
+			default: '#FEF9F3',
+		},
+	},
+	staticAttributes: ['Floor'],
+	preview: 'pet_bowl_preview.png',
+	slots: {},
+	pivot: {
+		x: 0,
```

Comment:
You forgot to set a pivot, which puts the icon to move the asset at an unexpected position.

In my tests, 
```
		x: 125,
		y: 50,
```
seemed like good values.